internal/services/dispatch/dispatchService_integration_test.go

```diff
@@ -0,0 +1,179 @@
+package dispatch_service
```

Comment:
Try to write integration test in separate package. So this well be like "black box testing" approach. Also this is better way to write integration tests in GO.

Reply:
Moved integration tests and its helpers to `internal/tests` directory